lib/eex/lib/eex/tokenizer.ex

```diff
@@ -37,15 +37,16 @@ defmodule EEx.Tokenizer do
   def tokenize(list, line, column, opts)
       when is_list(list) and is_integer(line) and line >= 0 and is_integer(column) and column >= 0 and
              is_list(opts) do
-    tokenize(list, line, column, opts, [], [])
+    opts = Keyword.put_new(opts, :indent, 0)
```

Comment:
Let's use the opportunity and make it a map. Only :trim and :indent seem to be used.